Tools/NTupleReader.cc

```diff
@@ -22,7 +22,7 @@ NTupleReader::NTupleReader(TTree * tree)
 void NTupleReader::init()
 {
     //gROOT->ProcessLine(".L TupleDict.h+");
-    gInterpreter->GenerateDictionary("vector<TLorentzVector>","TLorentzVector.h;vector");
+    //gInterpreter->GenerateDictionary("vector<TLorentzVector>","TLorentzVector.h;vector");
 
```

Comment:
Only create the autoDict. Can also achieve by pragma in the preprocessor.  This is my local change. Please ignore.